config/clusters/iam.tf

```diff
@@ -297,6 +297,84 @@ data "aws_iam_policy_document" "test-infra_s3_access" {
   }
 }
 
+# Falco repository (dev packages)
+
+module "falco_dev_s3_role" {
+  source    = "terraform-aws-modules/iam/aws//modules/iam-github-oidc-role"
+  version = "5.10.0"
+  name = "github_actions-falco-dev-s3"
+  create = true
+  subjects = [
+    "falcosecurity/falco:ref:refs/heads/master"
+  ]
+  policies = {
+    falco_s3_access = "${aws_iam_policy.falco_dev_s3_access.arn}"
+  }
+}
+
+resource "aws_iam_policy" "falco_dev_s3_access" {
+  name_prefix = "github_actions-falco-dev-s3"
+  description = "GitHub actions S3 access policy for falco repo dev workflows"
+  policy      = data.aws_iam_policy_document.falco_dev_s3_access.json
+}
+
+data "aws_iam_policy_document" "falco_dev_s3_access" {
+  statement {
+    sid    = "UploadFalcoDevS3Access"
+    effect = "Allow"
+    actions = [
+      "s3:PutObject",
+      "s3:GetObjectAcl",
+      "s3:GetObject",
+      "s3:DeleteObject",
+      "s3:PutObjectAcl"
+    ]
+    resources = [
+      "arn:aws:s3:::falco-distribution/packages/*-dev/*",
```

Comment:
I am not sure whether this regex is fine, but i think so.
Basically, master workflow can only PUT packages to the `dev` packages buckets.

Reply:
I'll try to check if this can work, I'm not sure either

Reply:
Turns out I didn't have an env set up for this test, but according to the documentation we're fine: https://docs.aws.amazon.com/AmazonS3/latest/userguide/s3-arn-format.html

Reply:
Yep that's what i read too!